### DISCUSSION TITLE
Content message is returning empty

### DISCUSSION THREAD

**Discussion by osgyn** [discord-php/DiscordPHP#1054] [Q&A] [answered]
**Environment**
- PHP Version:
    - 8.1.5
- DiscordPHP Version:
    - 7.2.6

**Describe the bug**
After ~11:50 UTC all my messages is returning with content empty

**To Reproduce**
```php
$discord->on('ready', function ($discord) {
    echo 'Bot is ready!' . PHP_EOL;
    $discord->on('message', function ($message, $discord) {
        var_dump($message->content);
    });
});
```
Response: string(0) ""

**Expected behavior**
Content message

**Additional context**
The problem started today, before that my bot was running normally with ~2 months of uptime.

**Comment by osgyn** [comment]
I've found out.

Discord updated intents so I needed to generate authorization again with new permission "message content intent"

https://stackoverflow.com/a/73037243

**Comment by SQKo** [comment, marked as answer]
If your bot is not verified and can still switch the message content intent in the developers portal, you do not need to update and can stay in DiscordPHP v7 (API v9) until next year/until DiscordPHP v10 (API v10) is released.

For any verified Bots that do not have Message Content intent approved or want to take advantage of v10 API, please use version [`dev-master#7702c55`](https://github.com/discord-php/DiscordPHP/tree/7702c55861386fde5f2ef90bbc94e36cd8d0b30d). Please note this is currently not a stable release version, and already have some breaking change content with the removal of fields/constants.
